packages/p2p-v2/src/components/index.ts

```diff
@@ -1,14 +1,26 @@
 export * from './AdvertiserName';
 export * from './Badge';
 export * from './Button';
+export * from './Checklist';
 export * from './CloseHeader';
+export * from './Dropdown';
+export * from './FlyoutMenu';
 export * from './FullPageMobileWrapper';
+export * from './Input';
 export * from './MobileTabs';
 export * from './Modals';
```

Comment:
```suggestion
export * from './Modals';
```
Also remove this line to highlight error, so other ppl import from `@/components/Modals